Add pkg-config support to stable branch 1.3

aquette opened the pull request.

maekitalo committed:
fix freeing pooled mysql connection when prepareCached is used

Commit:
Added pkg-config support

karolhrdina committed:
Merge pull request #1 from aquette/1.3

Added pkg-config support

jimklimov committed:
Add a basic .travis.yml

jimklimov committed:
cache: ccache

jimklimov committed:
Dep on packaged cxxtools

jimklimov committed:
Dep on libcxxtools-dev instead

jimklimov committed:
Tntnet and Tntdb seem to ignore passed compilers; Use travis C++ presets

jimklimov committed:
Add a .gitignore

maekitalo commented:
Hi, I would prefer not to update 1.3 branch but to release 1.4 soon. Is that ok for you? In 1.4 is pkg-config support already available.

jimklimov committed:
.travis.yml : Cause use of newer gcc-4.8+

jimklimov committed:
Rename legacy configure.in to modern configure.ac

jimklimov committed:
configure.ac : do not name non-numeric version "1.3rc3" - this upsets pkg-config

aquette commented:
@maekitalo: 1.4 release would be great and very much appreciated! @jimklimov: anything missing in 1.4 from your pov?

jimklimov commented:
Sorry, no idea at the moment - our project runs with a fork of 1.3 release, with a few changes on top that can be seen in git, and AFAIK we did not investigate if we need/conflict the code in git head / 1.4 candidate. That would need some story points ;)

jana-rapava committed:
Problem: Tntnet crashes if DB stops responding.

Solution: Set connect timeout lower then TCP/IP Close_Wait_Timeout (10 minutes).

Signed-off-by: Jana Rapava <janarapava@eaton.com>

jana-rapava committed:
Problem: Tntnet crashes if DB stops responding.

Solution: Set connect timeout lower then TCP/IP Close_Wait_Timeout (10 minutes).

Signed-off-by: Jana Rapava <janarapava@eaton.com>

jimklimov committed:
Merge pull request #2 from jana-rapava/1.3

Problem: Tntnet crashes if DB stops responding.

jimklimov committed:
Merge branch '1.3' into 1.3-FTY
